serialization.rst

```diff
@@ -31,7 +33,10 @@ These serialize as a BSON document::
 
   [ 0 => 1, 2 => 8, 3 => 12 ] => { "0" : 1, "2" : 8, "3" : 12 }
   [ "foo" => 42 ] => { "foo" : 42 }
-  [ 1 => 9, 0 => 10 ] => { "1" : 9, "0" => 10 }
+  [ 1 => 9, 0 => 10 ] => { "1" : 9, "0" : 10 }
+
+Note that the five examples are *extracts* of a full document, and represent
+only **one** value inside a document.
```

Comment:
Is this language meant to support "A top-level (root) document, **always** serializes as a BSON document."?